pkg/arm/data/master-startup.sh

```diff
@@ -486,7 +488,11 @@ spec:
     - master
     - api
     - --config=/etc/origin/master/master-config.yaml
+{{- if not .Derived.RunningUnderTest }}
```

Comment:
I think I'd prefer the logic swapped on this.  That way the default case is "normal".
```
{{- if .Derived.RunningUnderTest }}
    - --loglevel=4
{{ - else }}
    - --loglevel=2
{{ - end }}
```

Thoughts?

Reply:
sorry, had to repush since I cp'd the above stanza and it has some extra whitespace.